packages/babel-preset-udemy-website/tests/result/spread.js

```diff
@@ -1,3 +1,4 @@
-'use strict';
+"use strict";
```

Comment:
We use a plugin to add `use strict`. This, apparently puts it above the imports. It doesn't matter. Webpack takes care of this stuff in practice.